miqa/learning/nn_classifier.py

```diff
@@ -211,7 +211,7 @@ def get_model(file_path=None):
 
     if file_path is not None:
         model.load_state_dict(torch.load(file_path, map_location=device))
-        print(f'Loaded NN model from file "{file_path}"')
+        # print(f'Loaded NN model from file "{file_path}"')
```

Comment:
Can we make this output (and a few more below) only active in development mode instead of commenting them out?

Reply:
I'd suggest we use the logging system rather than `print`, and set these at `debug` level if that's something we want.

Reply:
Is that active by default in PyCharm? If not, how do I enable it?

Reply:
I think the logger pattern is better than using `if` and `print`, since it handles this conditional concept internally.

Reply:
I don't see that we've implemented a logger anywhere. Should we set it up?

Reply:
The nice thing is, it's already set up! At the top of the file, just do:

```python
import logging

logger = logging.getLogger(__name__)
```

Then you can replace all the print statements `logger.debug`.

Reply:
Uncommented and replaced all print statements.